cmd/root.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright © 2024 Metal toolbox authors <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/metal-toolbox/bioscfg/internal/model"
+	"github.com/spf13/cobra"
+)
+
+const (
+	VarNameConfiguration = "config"
```

Comment:
I find prefixing `VarName` here distracting more than helpful. Also, there's no reason to export these values outside the module.

Reply:
Done